autochdir.kak

```diff
@@ -0,0 +1,38 @@
+##
+## autochdir.kak by wheatdog
+## Automatically change server's working directory according to the current focus buffer
+## To turn on this option, add `set global autochdir true` in your kakrc
+##
+
+decl bool autochdir
+decl str working_folder
+
+def -hidden \
+    autochdir-wrapper %{ %sh{
+        if [ "${kak_opt_autochdir}" == "true" ] && [ ! -z "${kak_opt_working_folder}" ] && [ -d "${kak_opt_working_folder}" ]; then
```

Comment:
String comparison should be done with a single `=`, `! -z` can be replaced by `-n`.